Makefile

```diff
@@ -3,3 +3,4 @@
 
```

Comment:
do we need this file still?

Reply:
I didn't add the file, I just added the line to modify the python files. But I agree with the fact that it is not really being used. Let me know if you want me to delete it